tests/unit/lms/services/organization_service_test.py

```diff
@@ -279,6 +279,24 @@ def test_usage_report(self, svc, org_with_parent, h_api):
         ]
         assert report == Any.list.containing(expected)
 
+    def test_usage_report_with_no_courses(self, svc, org_with_parent):
+        since = datetime(2023, 1, 1)
+        until = datetime(2023, 12, 31)
+
+        with pytest.raises(ValueError):
```

Comment:
Since `ValueError` is a very general exception, it would be good to assert on relevant contents of the message. Same for the other tests. Otherwise the test could still pass despite the wrong error being shown to the user.

Reply:
Coverage would assure that overall all the exception are tested but yeah this test will individually be testing the wrong thing.

Added some light asserting of the message value :+1: